src/app/landing/LandingPage.jsx

```diff
@@ -65,26 +65,36 @@ class LandingPage extends Component {
               <source
                 className="OnboardingImage"
                 media="(min-aspect-ratio: 2/3)"
-                srcSet="img/landing_page/map_tablet.png"
+                srcSet="img/landing_page/step1.png"
               />
               <img
                 className="OnboardingImage"
-                src="img/landing_page/map_narrow.png"
+                src="img/landing_page/step1_mobile.png"
```

Comment:
In the video, this map (and maybe a couple other places) seem a bit blurry. Is that just in the video, or does the resolution/quality for the mobile versions need to be improved?
If the latter, maybe Di could help.

Reply:
It's the latter, these were the images provided for mobile resolutions. I sent her a message, maybe we can find a middle ground.